### ISSUE TITLE
should use BDD test framework for node sdk

### ISSUE THREAD

**Issue by christo4ferris** [hyperledger-archives/fabric#1864]
## Description

should use BDD test framework such as [Jasmine](http://jasmine.github.io/) for node sdk
## Describe How to Reproduce

run the tests

**Comment by christo4ferris** [comment]
@smithbk @bmos299

**Comment by bmos299** [comment]
The team is adding more unit tests for the node sdk by adding to the fabric/sdk/node/unit folder.  @rameshthoomu is working on including the make node-sdk-unit-tests into the CI and is currently waiting on PR #2312 to be accepted.   @smithbk started using tape.    I have read positive things on tape relative to Jasmine at the following site, http://www.macwright.org/2014/03/11/tape-is-cool.html.   The advantage with Jasmine is its hooks into BDD.  However, is it a big deal if the testing occurs through the make process instead of BDD?  In either case, failures are flagged and presented to the submitter via the CI.    Thoughts are appreciated.

**Comment by smithbk** [comment]
I agree with you Barry.  From my perspective, the only advantage of a BDD tool is that it allows a business person to more easily read and write tests, which I think seldom happens in practice.

**Comment by angrbrd** [comment]
@christo4ferris have we decided what to do with this issue? Should we move this over to Jira? Or close? Thanks.
